internal/pkg/armeria/command-handlers.go

```diff
@@ -211,11 +210,37 @@ func handleMoveCommand(ctx *CommandContext) {
 		walkDir = "down"
 		arriveDir = "above"
 	default:
-		ctx.Player.client.ShowText("That's not a valid direction to move in.")
+		ctx.Player.client.ShowColorizedText("That's not a valid direction to move in.", ColorError)
 		return
 	}
 
 	newRoom := Armeria.worldManager.RoomInDirection(ctx.Character.Room(), d)
+
+	areaLink := ctx.Character.Room().Attribute(d)
+	if areaLink != "" {
+		na := strings.Split(areaLink, ",")
+		if len(na) != 4 {
+			ctx.Player.client.ShowColorizedText("Area Link Error - Not enough arguments.", ColorError)
+			return
+		}
+		fmt.Println(na)
+		if na != nil {
```

Comment:
I don't think `na` will ever be nil? Or did you see a situation where it was `nil`? My reasoning there is that `areaLink` will at the very least always be an empty string. And doing `strings.Split()` on an empty string will just result in a string slice of length 1.